conductor/lib/common.py

```diff
@@ -509,6 +509,8 @@ def get_config_file_paths(self, config_file=None):
             possible_paths = [x for x in os.environ['CONDUCTOR_CONFIG'].split(path_separator) if len(x) > 0]
```

Comment:
not part of this commit, but see `os.path.pathsep` for cross platform path separator goodness

Reply:
Tracking as https://conductorio.atlassian.net/browse/CR-1051